manifests/c/com/github/polymeilex/neothesia/0.2.1/com.github.polymeilex.neothesia.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: com.github.polymeilex.neothesia

```

```suggestion
PackageIdentifier: PolyMeilex.Neothesia
```
This should follow the standard format of Publisher.Package
After changing this, the manifest files will need to be moved and renamed